pkg/storegateway/gateway_blocks_http.go

```diff
@@ -156,9 +197,105 @@ func (s *StoreGateway) BlocksHandler(w http.ResponseWriter, req *http.Request) {
 		ShowDeleted: showDeleted,
 		ShowSources: showSources,
 		ShowParents: showParents,
+		ActionType:  action,
+		InfoText:    infoText,
 	}, blocksPageTemplate, req)
 }
 
+func (s *StoreGateway) BlocksWriteHandler(w http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	tenantID := vars["tenant"]
+	if tenantID == "" {
+		util.WriteTextResponse(w, "Tenant ID can't be empty")
+		return
+	}
+
+	if err := req.ParseForm(); err != nil {
+		util.WriteTextResponse(w, fmt.Sprintf("Can't parse form: %s", err))
+		return
+	}
```

Comment:
What's the status code of these responses? Looks like they should be at least a 400, but method call says  just "write text response".